composer.json

```diff
@@ -114,19 +114,24 @@
     "symfony/polyfill-php73": "^1.10",
     "symfony/swiftmailer-bundle": "^3.1",
     "symfony/symfony": "^3.4.37",
-    "tecnickcom/tcpdf": "^6.2.12",
+    "tecnickcom/tcpdf": "dev-main",
```

Comment:
Are you sure about this? Shouldn't we wait for a proper release?

Reply:
Yup, it is the branch we have fixed on our fork : https://github.com/PrestaShop/TCPDF/pull/1

I discussed with @atomiix about a dedicated branch to avoid conflicts but it doesn't look necessary. 
Maybe cause the fix should be applied soon to the library :thinking:  (https://github.com/tecnickcom/TCPDF/pull/510)

Reply:
It's like a dead project!
Really boring to contribute on tcpdf.

Reply:
Hmmm ... Sorry but I don't understand the point.

Do you ask to create another branch ? A tag ? Should we just wait and let the bug ?

Reply:
I think it's better to wait.
Idk technically is it normal to use a fork branch of tcpdf on PS or not?
Because if we had, you'll have my guarantee to fix all RTL problems just in a second :)

Reply:
@PrestaShop/prestashop-maintainers and especially @eternoendless , do you have an opinion about this ?

I think, in some cases we won't be able to just wait but Idon't know about the conditions/limit forking our dependencies. I don't think it would be a good idea to allow our forks diverging from original projects ^^' ...

Reply:
@eternoendless we're already using 6.4.4 in 8.0.x and 6.3.5 in 1.7.8.x, I think it's okay to update to 6.4.4.

It's only a naming convention for the branch name, right? I created the branch 6.4.4-patch anyway.

Reply:
@atomiix 

> we're already using 6.4.4 in 8.0.x

Yes but this PR is targeting 1.7.8.x

> It's only a naming convention for the branch name, right?

Not just. If we ever need to supply two different patches (one for 1.7.8.x and another for 8.x based on different versions), we'll need two different branches. Not having the main branch diverge from the original repo will also help if we need to sync forks, etc.